infrastructure/index.ts

```diff
@@ -17,6 +18,7 @@ const fullDomain = `${subDomain}.${domain}`;
 const certificateArn = cfg.require("certificateArn");
 const canonicalUserId = aws.s3.getCanonicalUserId({});
 const productionCanonicalId = canonicalUserId.then(user => user.id);
+const repoDefaultDescription = "Pulumi’s modern infrastructure as code platform empowers cloud engineering teams to work better together to ship faster using the world’s most popular programming languages. Pulumi works with AWS, Kubernetes, and over 50 cloud infrastructure providers."
```

Comment:
```suggestion
const repoDefaultDescription = "Pulumi’s modern infrastructure as code platform empowers cloud engineering teams to work better together to ship faster using the world’s most popular programming languages. Pulumi works with AWS, Kubernetes, and over 150 cloud infrastructure providers."
```

We now seem to use 100+ or even 150+ for this, for example on https://www.pulumi.com/product/ 😄